Analysis/Core/include/AnalysisCore/HFSelectorCuts.h

```diff
@@ -0,0 +1,133 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef HF_SELECTOR_CUTS_H_
+#define HF_SELECTOR_CUTS_H_
+#include <vector>
+#include <string>
+namespace o2::analysis
+{
+namespace pdg
+{
+enum code {
+  kD0 = 421,
+  kD0bar = -421
+};
+}
+
+// accounts for the offset so that pt bin array can be used to also configure a histogram axis
+template <typename C, typename T>
```

Comment:
Are you sure this works in case the lower edge of the first bin is not zero?

Reply:
Indeed, this currently does not account for lower bin edge. There are also configurables for the highest and lowest allowed candidate pT. I'll replace this with a better algorithm anyway, since this is called in loops and needs to be faster.